sc/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp

```diff
@@ -4,38 +4,36 @@
 <%@ page contentType="text/html;charset=UTF-8" %>
 
 <form action="search" id="home-search-atlas-form">
-    <h3>Genes search</h3>
+    <h3>Gene search</h3>
 
     <!-- No need to enclose in row as the component already uses Foundation classes -->
     <div id="target"></div>
 
-    <div class="row">
-        <div class="small-12 columns small">
-            Examples: <a href='${pageContext.request.contextPath}/search?geneQuery=[{"value":"REG1B"}]'>REG1B</a>,
-            <a href='${pageContext.request.contextPath}/search?geneQuery=[{"value":"zinc finger"}]'>zinc finger</a>,
-            <a href='${pageContext.request.contextPath}/search?conditionQuery=[{"value":"lung"}]'>lung</a>,
-            <a href='${pageContext.request.contextPath}/search?conditionQuery=[{"value":"leaf"}]'>leaf</a>,
-            <a href='${pageContext.request.contextPath}/search?conditionQuery=[{"value":"valproic acid"}]'>valproic acid</a>,
-            <a href='${pageContext.request.contextPath}/search?conditionQuery=[{"value":"cancer"}]'>cancer</a></label>
-        </div>
+    <div class="row column small padding-bottom-medium">
+        Examples: <a href='${pageContext.request.contextPath}/search?geneQuery=[{"value":"REG1B"}]'>REG1B</a>,
+        <a href='${pageContext.request.contextPath}/search?geneQuery=[{"value":"zinc finger"}]'>zinc finger</a>,
+        <a href='${pageContext.request.contextPath}/search?conditionQuery=[{"value":"lung"}]'>lung</a>,
```

Comment:
This is out of scope for this PR, but these `geneQuery` and `conditionQuery` URL parameters are [rather non-standard](https://stackoverflow.com/questions/2366260/whats-valid-and-whats-not-in-a-uri-query). The usual way would be `/search?gene=zinc%20finger`. Or if we want to pass JSON as a parameter, we should put it in the body of a POST request.